tensorboard/uploader/proto/experiment.proto

```diff
@@ -29,6 +29,10 @@ message Experiment {
   // The number of bytes used for storage of tensors in this experiment,
   // across all time series, including estimated overhead. Output-only.
   int64 total_tensor_bytes = 9;
+  // The number of bytes used for storage of the contents of blobs in this
+  // experiment, across all time series, including estimated overhead.
+  // Output-only.
+  int64 total_blob_bytes = 10;
```

Comment:
Needs corresponding fieldmask update?

Reply:
Looks like so. cc @ericdnielsen for confirmation. If confirmed, I can send a follow-up PR to add that.

Reply:
I'm not too familiar with the fieldmaks.  But it sounds likely that its needed here.